packages/lib/src/components/paywall/Renderer.svelte

```diff
@@ -76,6 +76,12 @@
 
     loading = true;
 
+    if (product?.url && product?.url !== articleUrl) {
+      window.location.assign(product.url);
+      loading = false;
```

Comment:
Why are we setting loading to false here?

Reply:
we have had issues in the past where the loading state is stuck after a browser back navigation.. Browsers are trying to be smart and remember the javascript state so i just did it as a reflex.. But i will check if i can reproduce it otherwise i'll remove it

Reply:
removed it.